no-useless-escape should ignore line breaks in strings

**What version of ESLint are you using?**
2.5.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
Use this:
`/*eslint no-useless-escape: 2 */`

**What did you do? Please include the actual source code causing the issue.**
Create a multiline string separated by `\` like this:

```js
/* exported selectorCss */
function selectorCss() {
	return '.someclass {\
		color: #000000;\
		opacity: 0.8;\
	}';
}
```
**What did you expect to happen?**
No errors or warnings

**What actually happened? Please include the actual, raw output from ESLint.**
```sh
  5:9  error  Unnecessary escape character: \
  no-useless-escape
  5:9  error  Unnecessary escape character: \
  no-useless-escape
  5:9  error  Unnecessary escape character: \
  no-useless-escape

✖ 3 problems (3 errors, 0 warnings)
```

--- a/lib/rules/no-useless-escape.js
+++ b/lib/rules/no-useless-escape.js
@@ -19,7 +19,9 @@ var VALID_STRING_ESCAPES = [
     "b",
     "f",
     "u",
-    "x"
+    "x",
+    "\n",
+    "\r"
 ];
 
 var VALID_REGEX_ESCAPES = [

--- a/tests/lib/rules/no-useless-escape.js
+++ b/tests/lib/rules/no-useless-escape.js
@@ -43,7 +43,9 @@ ruleTester.run("no-useless-escape", rule, {
         "var foo = '\\n';",
         "var foo = 'foo \\r bar';",
         "var foo = '\\v';",
-        "var foo = '\\f';"
+        "var foo = '\\f';",
+        "var foo = '\\\n';",
+        "var foo = '\\\r\n';"
     ],
 
     invalid: [

